plugin.xml

```diff
@@ -127,14 +91,27 @@
     <resource-file src="src/ios/resources/Images.xcassets" />
     <resource-file src="src/ios/resources/SalesforceSDKAssets.xcassets" />
 
-    <!-- Libraries -->
-    <source-file src="src/ios/frameworks/libSalesforceSDKCommon.a" framework="true" />
-    <source-file src="src/ios/frameworks/libSalesforceAnalytics.a" framework="true" />
-    <source-file src="src/ios/frameworks/libSalesforceSDKCore.a" framework="true" />
-    <source-file src="src/ios/frameworks/libSmartStore.a" framework="true" />
-    <source-file src="src/ios/frameworks/libSmartSync.a" framework="true" />
-    <source-file src="src/ios/frameworks/libSalesforceHybridSDK.a" framework="true" />
-    <source-file src="src/ios/frameworks/libsqlcipher.a" framework="true" />
+    <!-- Mobile SDK frameworks pod specs -->
+    <pods-config ios-min-version="11.0" use-frameworks="true" />
```

Comment:
Pod specs for our libraries. See my comments below. We need to remember to flip this from `dev` to versioned tag as part of the release process.